pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -78,16 +85,16 @@ type InTreeToCSITranslator interface {
 // 2. Once all the assume operations are done in d), the scheduler processes the next Pod in the scheduler queue
 //    while the actual binding operation occurs in the background.
 type SchedulerVolumeBinder interface {
-	// FindPodVolumes checks if all of a Pod's PVCs can be satisfied by the node.
+	// FindPodVolumes checks if all of a Pod's volumes can be satisfied by the node.
```

Comment:
change comment back to PVC since that's the current behavior.

Reply:
Okay.